docs/_sidebar.md

```diff
@@ -3,6 +3,6 @@
 * [Home](/)
 * [Quick Start](quick-start.md)
 * [Interact with Drava](user-interactions.md)
-* [Development Mode](dev.md)
 * [Use Cases](use-cases.md)
+* [Development Mode](dev.md)
```

Comment:
Making the interaction and use cases sections closer together seemed better.

Reply:
Make sense to me! Thanks